packages/react/.gemini.js

```diff
@@ -3,7 +3,7 @@ const dotenv = require("dotenv");
 dotenv.config({ path: "./gemini/.env" });
 
 module.exports = {
-  rootUrl: "http://localhost:9001",
+  rootUrl: "http://localhost:9876",
```

Comment:
Changing ports so that you can run tests without having to kill your dev storybook.